src/components/core/I13nComponent.jsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2015 - Present, Yahoo Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+
+import { useEffect } from 'react';
+
+import useScanLinks from '../../hooks/useScanLinks';
+
+const I13nComponent = (props) => {
+
+};
+
+export default I13nComponent;
```

Comment:
This component will be the main class handling different functions (replacing `ComponentSpecs`)